debug.log

```diff
@@ -2,3 +2,15 @@
 [0202/071721.062:ERROR:settings.cc(320)] Settings magic is not 1129342067
 [0225/093236.841:ERROR:settings.cc(320)] Settings magic is not 1129342067
 [0316/115825.945:ERROR:file_io.cc(134)] WriteFile: The pipe is being closed. (0xE8)
+[0523/232541.901:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
```

Comment:
Can you please remove this file. It is not relevant to the code I believe but seems to be log files from your local development server

Reply:
okay, i will remove it.